### ISSUE TITLE
Auto-mounted partitions

### ISSUE THREAD

**Issue by Sadi58** [feren-OS/Bug-Reporting-Center#98]
**Package Name or Section of Feren OS where this bug resides**
Nemo File Manager?

**Describe the bug**
After mounting a partition in the same disk as Feren OS is installed, it is auto-mounted at the next system startup even though it is not included in fstab, and even though it has been unmounted afterwards.

**Expected behavior**
No partition in the fixed disk (other than removable media as specified in nemo file manager preferences) should be auto-mounted unless it is included in fstab.

**Comment by feren** [comment]
That's probably Plasma doing automatic mounting of previously mounted drives. You can manage this in System Settings -> Removable Storage -> Removable Devices.

**Comment by Sadi58** [comment]
Thanks! I remember now that I had left KDE years ago for such reasons: many many bugs and weirdly working settings although its GUI provides many more settings and definitely looks much more appealing. ;-)
I decided to give it another try now via Feren OS which is quite promising as I see it.
When I looked at System Settings -> Removable Storage -> Removable Devices none of my partitions were checked actually, but checking only 3 partitions (which are auto-mounted on login anyway via fstab) and making sure that the first two options under "Enable automatic mounting of removable media" (enabled) are not checked did fix my problem - although these partitions are not removable media. ;-)
